### PR TITLE
Adding Biegel coal-to-nuclear GAIN pres.

### PR DESCRIPTION
**Pull request by biegelk**
Adding Biegel presentation for the coal-to-nuclear GAIN webinar to the Presentations page.

The slides will be made publicly available soon, but haven't been released yet. I'll make a separate PR to link the slides when they become available.

### CODE REVIEW

File: _data/presentations.yml [hunk 1]
```diff
@@ -4,4 +4,11 @@
...
   person: Paul Wilson
-  person_dir: pphw
\ No newline at end of file
+  person_dir: pphw
+
+- date: "2022-10-04"
+  venue: Gateway for Accelerated Innovation in Nuclear (GAIN)
+  venue_url: https://gain.inl.gov/SitePages/Home.aspx
+  title: "Investigating Benefits and Challenges of Converting Coal Power Plants to Nuclear Power Plants"
+  person: Katie Biegel
```
Comment (gonuke) [inline _data/presentations.yml:13]: How about adding this as a `url:` https://gain.inl.gov/SitePages/Coal2Nuclear.aspx?

**Review by gonuke — CHANGES_REQUESTED** [review]
Just recommending a URL for the existing landing page at GAIN.

File: _data/presentations.yml (hunk 1 above)
Comment (biegelk) [reply _data/presentations.yml:14]: Makes sense! Added

**Review by gonuke — APPROVED** [review]
Thanks @biegelk

**Pull request merged**